libs/gi/good/src/IArtifact.ts

```diff
@@ -10,10 +10,13 @@ import type {
 // GOOD 3 and above
 export interface IArtifact extends IBaseArtifact {
   totalRolls?: number // 3-9 for valid 5* artifacts; includes starting rolls
-  startingRolls?: number // 3-4 for valid 5* artifacts; If a 4th substat is in substats array, and this value is 3, the 4th substat is a revealed stat, but should not be factored into calculation
   astralMark?: boolean // Favorite star in-game
-  discard?: boolean // Trash icon in-game
-  elixirCrafted?: boolean // Flag for if the artifact was created using Sanctifying Elixir. This guarantees the main stat + 2 additional rolls on the first 2 substats
+  elixirCrafted?: boolean // Flag for if the artifact was created using Sanctifying Elixir. This guarantees the main stat + at least 2 rolls on the first 2 substats
+  unactivatedSubstats?: ISubstat[] // Unactivated substat(s). Once a substat is activated, it should be moved to `substats` instead
```

Comment:
Should this be `IBaseSubstat` since `initialValue` does not make sense for an unactivated substat?

Reply:
The initial value is known for unactivated substats technically